html/index.html

```diff
@@ -189,7 +187,15 @@
       hljs.highlightBlock(block);
     });
   })
-});
+};
+
+var contents = null;
+//INSERT_CONTENTS_HERE//

```

Comment:
May be an other anchor like `BECHAMEL` or `BECHAMEL_CONTENTS` and I will be happy.